<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QMainWindow" name="SecondWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1081</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="lab01">
     <attribute name="title">
      <string>Lab01</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>391</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Image 1</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLabel" name="image1label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>141</width>
          <height>121</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="image1_1table">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image1_2table">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image1_3table">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энергия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="enegry_1">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энтропия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="entropiya_1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>270</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>290</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Контраст:</string>
        </property>
       </widget>
       <widget class="QLabel" name="contrast_1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>290</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="homo_1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>310</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>310</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Гомогенность</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Image 2</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLabel" name="image2label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>131</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="image_2_1_table">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image2_2table">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image2_3table">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="enegry_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="contrast_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>290</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энергия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="entropiya_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>270</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энтропия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>290</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Контраст:</string>
        </property>
       </widget>
       <widget class="QLabel" name="homo_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>310</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>310</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Гомогенность</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Image 3</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLabel" name="image3label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="image3_1_table">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image3_2table">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image3_3table">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>40</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="enegry_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="contrast_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>290</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энергия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="entropiya_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>270</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энтропия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>290</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Контраст:</string>
        </property>
       </widget>
       <widget class="QLabel" name="homo_6">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>310</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>310</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Гомогенность</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Image 4</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLabel" name="image4label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>121</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="image4_1table">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image4_2table">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="image4_3table">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="enegry_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="contrast_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>290</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>250</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энергия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="entropiya_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>270</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>270</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Энтропия:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>290</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Контраст:</string>
        </property>
       </widget>
       <widget class="QLabel" name="homo_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>310</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>310</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Гомогенность</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>430</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>R12</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>450</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>R34</string>
      </property>
     </widget>
     <widget class="QLabel" name="r12">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>430</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="r34">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compareButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>480</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Compare</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lab02">
     <attribute name="title">
      <string>Lab02</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1061</width>
        <height>591</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QLabel" name="image2label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>50</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="image2open">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>95</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="thinnerTable">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>731</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="thinButton">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>350</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Be Thinner
Look Better</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
